wiki/Getting-Started-UIKit.md

```diff
@@ -0,0 +1,248 @@
+# Interested in seeing examples of Workflow in action?
+Start by cloning the repo and checking out the 'WorkflowExample' scheme. This should give you a decent idea of how the library works.  If you want to create a new project, read on.
+
+# CocoaPods with Programmatic Views
+
+## Adding the dependency
```

Comment:
We should have a conversation about what our preferred package manager is. Most of our instructions reference CocoaPods but it seems maybe more likely our preferred package manager should be SwiftPM.

Reply:
Discussion started: https://github.com/wwt/Workflow/discussions/40

Reply:
As this is a bigger discussion to have and there is an existing discussion linked.  I'm resolving this conversation, and it should continue in #40